emulator.c

```diff
@@ -269,6 +289,10 @@ execute_instruction(i8080 *cpu, uint8_t opcode)
       }
     case 0xc1: // NOLINT
       printf("POP B");
+      cpu->c = cpu->memory[cpu->sp];
+      cpu->b = cpu->memory[cpu->sp + 1];
+      cpu->sp += 2;
+      cpu->pc += 1;
```

Comment:
I don't think you need to increment pc here? 
![image](https://github.com/cliff-bielinski/space-invaders-8080-emulator/assets/122326493/b0d978f7-d17b-48f1-9791-b711b0344991)

Reply:
Agreed - it's a one opcode instruction.